### ISSUE TITLE
corpus_install() from Zenodo not working

### ISSUE THREAD

**Issue by ablaette** [PolMine/cwbtools#42]
This newly results in the error message "get Zenodo record referred to by DOI ... failed": 

```{r}
library(cwbtools)
corpus_install(doi = "10.5281/zenodo.3742113")
```

The error results internally from this sequence:

```{r}
library(zen4R)
z <- ZenodoManager$new()
z$getRecordByDOI(doi = "10.5281/zenodo.3742113")
```

Or, more specifically from:

```{r}
library(zen4R)
z <- ZenodoManager$new()
z$getRecords("doi:10.5281//zenodo.3742113")
```

And this is an issue that has beeen reported:  https://github.com/eblondel/zen4R/issues/84

Indeed, there is a pending pull request, but zen4R is currently not yet fixed (neither at GitHub nor at CRAN), so a workaround is required.

**Comment by ablaette** [comment]
There is a new version on the dev branch submitted to CRAN that uses a workaround utility function `zenodo_get_tarballurl()`. I would like to revert to the zen4R functionality as soon as it is works again. I keep this issue open to check the state of zen4R regularly.

**Comment by ablaette** [comment]
Update: The code above to reproduce the issue does not yet work again. See this issue to confirm that other uses encounter the issue too: https://github.com/eblondel/zen4R/issues/98

**Comment by eblondel** [comment]
@ablaette It's solved in zen4R 0.8 (github for now)
